src/components/testAngular/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
```

Comment:
Are these testAngular files automatically generated, or created by you?

Reply:
Autogenerated - everything inside the component is autogenerated!